### ISSUE TITLE
Destructured named exports don't work

### ISSUE THREAD

**Issue by rixo** [PepsRyuu/nollup#93]
(Test case on [this branch](https://github.com/rixo/nollup-test-case-output-dir/tree/destruct-named-exports))

As mentioned [earlier](https://github.com/PepsRyuu/nollup/issues/84#issuecomment-642123959), named exports in the following code don't work with Nollup:

~~~js
const foobar = { foo: 1, bar: 2 }

export const { foo, bar } = foobar
~~~

Given the following `main.js`:

~~~js
import { foo, bar } from './foobar.js'

console.log(foo, bar)
~~~

We get the following console log with Nollup:

~~~
undefined undefined
~~~

Whereas with Rollup:

~~~
1 2
~~~

**Comment by PepsRyuu** [comment]
Pushed fix to ```refactor``` branch.

**Comment by rixo** [comment]
I confirm this is fixed in Routify. I now have 2/3 projects that can run off the raw Refactor branch with no patches :tada:

Sapper still needs some bundling events (BUNDLE_START, BUNDLE_END... not yet investigated). Maybe more.

Interestingly enough, I'm now able to reproduce the `file` issue we've been discussing in #84 right on [the branch](https://github.com/rixo/nollup-test-case-output-dir/tree/destruct-named-exports) I had used to report the current issue. I've just updated it to install the Refactor branch, no other change.

The cause is the same as what I had seen with `inlineDynamicImports` (but this time the option is not used, just `output.file`): the entry point is written as an absolute path in the `files` object in dev middleware. And so it's not available at `/main.js`...

Am I missing something obvious?

**Comment by rixo** [comment]
Here too, commenting out the whole `if (output.file) ...` condition [here](https://github.com/PepsRyuu/nollup/blob/6315693d1335fec846f104c084a734915796d306/lib/dev-middleware.js#L87) fixes the problem for me. I haven't thought through the consequences...

So I can confirm the destructured exports problem is fixed in the test case too.

**Comment by PepsRyuu** [comment]
Released in ```0.12.0```
